SCR_FSW_Prometheus/SCR_FSW_Prometheus.ino

```diff
@@ -260,30 +246,32 @@ void loop(void)
         state = kRecovery; //Change state to Recovery
 
       }
+}
+
+void loop(void)
+{
+  switch (state) {
+    case kStandby: //Standby state code block
+      break;
+    case kAscent: //Ascent state code block
+      imuRetrieve();
+      bmpRetrieve();
+      Serial.println(a + packetno + b + a + altitude + b + a + pressure + b + a + temp + b + a + xaccel + b + a + yaccel + b + a + zaccel + b + a + tiltx + b + a + tilty + b + a + tiltz + b);
```

Comment:
Is this just for the serial monitor or is this meant for the XBEE? For the XBEE I think you should make a dedicated "transmit" function or some such. 

Also, lumping it all in one *.println() command doesn't look to great imo. It might be cleaner to use multiple .print() statements and just terminate it with a println or "\n"